testsupport/init.go

```diff
@@ -53,7 +53,7 @@ func WaitForDeployments(t *testing.T, obj runtime.Object) (*framework.TestCtx, *
 	require.NoError(t, err, "failed while waiting for member operator deployment")
 
 	// wait for registration service to be ready
-	err = e2eutil.WaitForDeployment(t, f.KubeClient, registrationServiceNs, "registration-service", 3, wait.DefaultRetryInterval, wait.DefaultOperatorTimeout)
+	err = e2eutil.WaitForDeployment(t, f.KubeClient, registrationServiceNs, "registration-service", 1, wait.DefaultRetryInterval, wait.DefaultOperatorTimeout)
```

Comment:
I think we need to change it two at least 2 replicas: https://github.com/codeready-toolchain/host-operator/blob/master/deploy/env/e2e-tests.yaml#L3
So we can have our tests with multiple reg-service replicas to make verify it's a stateless service and there is no any issues with running multiple replicas.

We can do it in separate PRs though.

Reply:
good point. But I can do it easily here. it's as simple as these two commits:
0483817 
https://github.com/codeready-toolchain/host-operator/pull/117/commits/0610d667fdc411e1eb974c98454f4ccc765771f2